src/iris/api.py

```diff
@@ -1737,6 +1753,33 @@ def on_post(self, req, resp):
         resp.set_header('Location', '/incidents/%s' % incident_id)
         resp.body = ujson.dumps(incident_id)
 
+        # optional incident handler to do additional tasks after the incident has been created
+        if self.custom_incident_handler_module is not None:
+            connection = db.engine.raw_connection()
+            cursor = connection.cursor(db.dict_cursor)
+
+            # get plan info
```

Comment:
this section smells like the same logic for query a plan. Can you abstract them away in another function?

Reply:
It's the same code but I don't think it's worth abstracting away since it's a pretty small snippet that isn't going to get reused anywhere else, and the end result would be slightly less readable. I just doesn't seem necessary especially since all the endpoints currently handle everything within their own class anyways.